Software/Python/Examples/Connectome/connectome.py

```diff
@@ -0,0 +1,5005 @@
+
+def CEPVR():
+        postSynaptic['ASGR'][nextState] += 1
+        postSynaptic['AVEL'][nextState] += 5
+        postSynaptic['IL1VR'][nextState] += 1
+        postSynaptic['IL2VR'][nextState] += 2
+        postSynaptic['MVR04'][nextState] += 1
+        postSynaptic['OLLR'][nextState] += 7
+        postSynaptic['OLQVR'][nextState] += 3
+        postSynaptic['OLQVR'][nextState] += 1
+        postSynaptic['RICL'][nextState] += 2
+        postSynaptic['RICR'][nextState] += 2
+        postSynaptic['RIH'][nextState] += 1
+        postSynaptic['RIPR'][nextState] += 1
+        postSynaptic['RIVL'][nextState] += 1
+        postSynaptic['RMDDR'][nextState] += 2
+        postSynaptic['RMHR'][nextState] += 2
+        postSynaptic['SIAVR'][nextState] += 2
+        postSynaptic['URAVR'][nextState] += 1
+
+def DA1():
+        postSynaptic['AVAL'][nextState] += 2
+        postSynaptic['AVAR'][nextState] += 6
+        postSynaptic['DA4'][nextState] += 1
+        postSynaptic['DD1'][nextState] += 4
+        postSynaptic['MDL08'][nextState] += 8
+        postSynaptic['MDR08'][nextState] += 8
+        postSynaptic['SABVL'][nextState] += 2
+        postSynaptic['SABVR'][nextState] += 3
+        postSynaptic['VD1'][nextState] += 17
+        postSynaptic['VD2'][nextState] += 1
+
+def DA2():
+        postSynaptic['AS2'][nextState] += 2
+        postSynaptic['AS3'][nextState] += 1
+        postSynaptic['AVAL'][nextState] += 2
+        postSynaptic['AVAR'][nextState] += 2
+        postSynaptic['DD1'][nextState] += 1
+        postSynaptic['MDL07'][nextState] += 2
+        postSynaptic['MDL08'][nextState] += 1
+        postSynaptic['MDL09'][nextState] += 2
+        postSynaptic['MDL10'][nextState] += 2
+        postSynaptic['MDR07'][nextState] += 2
+        postSynaptic['MDR08'][nextState] += 2
+        postSynaptic['MDR09'][nextState] += 2
+        postSynaptic['MDR10'][nextState] += 2
+        postSynaptic['SABVL'][nextState] += 1
+        postSynaptic['VA1'][nextState] += 2
+        postSynaptic['VD1'][nextState] += 2
+        postSynaptic['VD2'][nextState] += 11
+        postSynaptic['VD3'][nextState] += 5
+
+def DA3():
+        postSynaptic['AS4'][nextState] += 2
+        postSynaptic['AVAR'][nextState] += 2
+        postSynaptic['DA4'][nextState] += 2
+        postSynaptic['DB3'][nextState] += 1
+        postSynaptic['DD2'][nextState] += 1
+        postSynaptic['MDL09'][nextState] += 5
+        postSynaptic['MDL10'][nextState] += 5
+        postSynaptic['MDL12'][nextState] += 5
+        postSynaptic['MDR09'][nextState] += 5
+        postSynaptic['MDR10'][nextState] += 5
+        postSynaptic['MDR12'][nextState] += 5
+        postSynaptic['VD3'][nextState] += 25
+        postSynaptic['VD4'][nextState] += 6
+
+def DA4():
+        postSynaptic['AVAL'][nextState] += 3
+        postSynaptic['AVAR'][nextState] += 2
+        postSynaptic['DA1'][nextState] += 1
+        postSynaptic['DA3'][nextState] += 1
+        postSynaptic['DB3'][
```

Comment:
Does 27 even make sense? it's now 27 mm.  us_dist was returning values in cm, if memory serves me right.